pkg/compute/kube/types.go

```diff
@@ -1,22 +1,20 @@
 package kube
 
 import (
+	"fmt"
+
 	"github.com/openservicemesh/osm/pkg/k8s"
 	"github.com/openservicemesh/osm/pkg/logger"
 )
 
-const (
-	// providerName is the name of the Kubernetes client that implements service.Provider and endpoint.Provider interfaces
-	providerName = "kubernetes"
-)
-
 const (
 	// kindSvcAccount is the ServiceAccount kind
 	kindSvcAccount = "ServiceAccount"
 )
 
 var (
-	log = logger.New("kube-provider")
+	log                = logger.New("kube-provider")
+	errServiceNotFound = fmt.Errorf("Service not found")
```

Comment:
nit: errors shouldn't be capitalized

Reply:
fixed

Reply:
also switched to errors.New, which has less allocations :)

Reply:
Oh does it really? TIL